src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  res.send(userService.getAllUsers());
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
```

Comment:
You are checking if 'name' is provided after trying to create the user. This check should be done before attempting to create the user to avoid unnecessary calls to the 'createUser' function if the 'name' is not provided.